src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -27,6 +27,23 @@
 
 HOST_NAMESPACE_DB_IDX = 0
 
+RIF_COUNTERS_AGGR_MAP = {
```

Comment:
should we move the map to the file which use it? i see it is only used in one place.

Reply:
We don’t think so, as it seems to be the convention to define variables in __init__.py, even if they are only used in one other file.  For example, Qi Luo recently added HOST_NAMESPACE_DB_IDX which you see above, defined in __init__.py and used only in rfc3433.py.